tests/inputs/models/2022_inj_b1_adt/job.create_model_nominal.madx

```diff
@@ -0,0 +1,38 @@
+title, "LHC Model created by omc3";
+! Model directory: /afs/cern.ch/work/a/awegsche/public/forTuuli/omc3/tests/inputs/models/2022_inj_b1_adt
+! Natural Tune X         [     0.310]
+! Natural Tune Y         [     0.320]
+! Best Knowledge:        [        NO]
+! Excitation             [       ADT]
+! > Driven Tune X        [     0.298]
+! > Driven Tune Y        [     0.330]
+
+call, file = '/afs/cern.ch/work/a/awegsche/public/forTuuli/omc3/tests/inputs/models/2022_inj_b1_adt/macros/general.macros.madx';
```

Comment:
Looks like you're committing your test files that shouldn't be here, this even points to a local file in your computer...

Reply:
well, the test files should be there. all the madx scripts maybe not

Reply:
removed all .madx files